src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,12 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery winCar = new Lottery();
+        System.out.println(winCar.getRandomBall());
+        Lottery winWine = new Lottery();
+        System.out.println(winWine.getRandomBall());
+        Lottery winPizza = new Lottery();
+        System.out.println(winPizza.getRandomBall());
```

Comment:
Make `for-loop` for that logic.
Also you don't need to create a new Lottery to get a new random ball every time.